libs/vue-plugin/src/builders/dev-server/builder.ts

```diff
@@ -72,13 +98,14 @@ export function runBuilder(
           });
         }
       },
+      outputDir: getSystemPath(
```

Comment:
Why is this required?

Reply:
`modifyCopyAssets` copies the assets to `options.outputPath` so we either have to change where they are copied or specify the outputDir. I went with the latter because it's nice having the build and serve builders mirror each other.